hedera-node/src/main/java/com/hedera/services/context/BasicTransactionContext.java

```diff
@@ -223,10 +231,15 @@ public ResponseCodeEnum status() {
 	}
 
 	@Override
-	public TxnAccessor accessor() {
+	public SignedTxnAccessor accessor() {
 		return accessor;
 	}
 
+	@Override
```

Comment:
Here we can change the return type to the interface, `SwirldsTxnAccessor`, and use,
```suggestion
	@Override
	public SwirldsTxnAccessor platformTxnAccessor() {
		if (accessor instanceof SwirldsTxnAccessor swirldsTxnAccessor) {
			return swirldsTxnAccessor;
		} else {
			throw new IllegalStateException("This context did not originate from a Swirlds transaction");
		}
	}
```

Reply:
Fixed